corehq/apps/app_manager/templates/app_manager/partials/graph_configs.html

```diff
@@ -0,0 +1,47 @@
+{% load i18n %}
+
+<div data-bind="foreach: { data: graphConfig.currentCharts, as: 'chart' }">
+    <br/>
+    <p>
+        {% trans 'Chart Title' %}: <span data-bind="text: chart.title"></span>
+    </p>
+    <p>
+        <span>{% trans 'Graph type' %}:</span>
+        <select type="text" data-bind="
+            options: $parent.graphConfig.allGraphTypes,
+            value: $parent.graphConfig.getCurrentGraphConfig(chart.chart_id).graph_type
+        "></select>
+    </p>
+    <p>
+        {% trans 'Configuration' %}:
+        <div data-bind="foreach: { data: $parent.graphConfig.getCurrentGraphConfig(chart.chart_id).config.keyValuePairs, as: 'keyValuePair' }">
+            <p>
+                <input type="text" data-bind="value: keyValuePair[0]">
+                -->
```

Comment:
...definitely looked at this twice, thinking, "where's the beginning of the comment?"